definitions/npm/rxjs_v5.0.x/flow_v0.29.0-v0.32.x/rxjs_v5.0.x.js

```diff
@@ -23,6 +23,12 @@ interface rxjs$ISubscription {
 
 type TeardownLogic = rxjs$ISubscription | () => void;
 
+type EventListenerOptions = {
```

Comment:
Really need to namespace these global types.
Not your doing though, so if you don't want to do it in this PR that's fine and I can do it in a follow-up.